apisix/plugins/skywalking.lua

```diff
@@ -17,6 +17,7 @@
 local sw_tracer = require("skywalking.tracer")
 local core = require("apisix.core")
 local process = require("ngx.process")
+local Span = require('skywalking.span')
```

Comment:
I think it would be better to keep the style uniform.

```suggestion
local Span = require("skywalking.span")
```